### ISSUE TITLE
Research: Ensure secure access to backend

### ISSUE THREAD

**Issue by Jean28518** [amosproj/amos2022ss04-digital-identity#39]
## User story
1. As company
2. I need to ensure security in the backend
5. So that no unintended person can access the api.

## Acceptance criteria
* Concrete strategy has to be presented

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
